docs/metrics.md

```diff
@@ -0,0 +1,103 @@
+# Backbeat Metrics
+
+Backbeat exposes various metric routes which return a response with an HTTP
+code.
+
+## Response Codes
+
+```
++----------+------------------------------------------------------------------+
+| Response | Details                                                          |
++==========+==================================================================+
+|   200    | OK: success                                                      |
++----------+------------------------------------------------------------------+
+|   403    | AccessDenied: request IP address must be defined in              |
+|          |   'conf/config.json' in field 'server.healthChecks.allowFrom'    |
++----------+------------------------------------------------------------------+
+|   404    | RouteNotFound: route must be valid                               |
++----------+------------------------------------------------------------------+
+|   405    | MethodNotAllowed: the HTTP verb must be a GET                    |
++----------+------------------------------------------------------------------+
+|   500    | InternalError: this could be caused by one of several            |
+|          |   components: the api server, Kafka, Zookeeper, Redis, or one    |
+|          |   of the Producers for a topic topic                             |
++----------+------------------------------------------------------------------+
+```
+
+## Routes
+
+Routes are organized in the following fashion:
+`/_/metrics/<extension-type>/<site-name>/<metric-type>`
+
+Where:
+
+- `<extension-type>` can currently only support `crr` for replication metrics
+- `<site-name>` represents any current sites you have defined. If you would like
+  metrics displayed for all sites, use `all`
+- `<metric-type>` is an optional field. Leaving this out will get all metrics
+  available for given extension and site. If you specify a metric type, you will
+  get the metric specified.
+
+### `/_/metrics/<extension-type>/<site-name>`
```

Comment:
The following metrics are probably bound to CRR for now, they will likely not apply to other extensions, so you can be specific to CRR in your description (and embed them in a "CRR" chapter)